client/src/App.js

```diff
@@ -1,19 +1,30 @@
 import './App.css';
 
-import React, { useState } from 'react';
+import {
+  Redirect,
+  Route,
+  BrowserRouter as Router,
+  Switch,
+} from 'react-router-dom';
 
-import { HEADERS, PAGE_MAP } from './constants/header_constants';
 import PageNav from './components/PageNav/PageNav';
+import React from 'react';
+import Search from './components/Search/Search';
+import SentimentAnalysis from './components/SentimentAnalysis/SentimentAnalysis';
+import Visualization from './components/Visualization/Visualization';
 
-const App = () => {
-  const [currentPage, setCurrentPage] = useState(HEADERS[0]);
-
-  return (
+const App = () => (
+  <Router>
```

Comment:
So I assume that this is now a multipage webapp and you've added the router to switch between pages. Will this change have any impact on the website as we've built it thus far?

Reply:
Nah it should be the same for everything else!